Libraries/LibWeb/SVG/SVGScriptElement.cpp

```diff
@@ -40,15 +43,75 @@ void SVGScriptElement::process_the_script_element()
     if (m_already_processed || !in_a_document_tree())
         return;
 
-    auto inline_script = child_text_content();
+    IGNORE_USE_IN_ESCAPING_LAMBDA String script_content;
+    auto script_url = m_document->url();
 
-    // FIXME: 2. If the 'script' element references external script content, then the external script content
-    //           using the current value of the 'xlink:href' attribute is fetched. Further processing of the
-    //           'script' element is dependent on the external script content, and will block here until the
-    //           resource has been fetched or is determined to be an invalid IRI reference.
+    // 2. If the 'script' element references external script content, then the external script content
+    //    using the current value of the 'xlink:href' attribute is fetched. Further processing of the
+    //    'script' element is dependent on the external script content, and will block here until the
+    //    resource has been fetched or is determined to be an invalid IRI reference.
```

Comment:
Goodness the spec ref at the top of this function is ancient.

We should try to follow this, I presume?

https://www.w3.org/TR/SVG2/linking.html#processingURL-parsing
https://www.w3.org/TR/SVG2/linking.html#processingURL-fetch